<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_slot">
     <property name="title">
      <string>Slot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_slot_warehouse">
        <property name="text">
         <string>Warehouse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_slot_warehouse"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Slot:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_slot">
        <property name="placeholderText">
         <string>Enter slot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_product">
     <property name="title">
      <string>Product</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_product_sku">
        <property name="text">
         <string>SKU:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_product_sku</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_product_sku">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_product_lookup">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Lookup</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_product_description">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_product_description</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_product_description">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Enter SKU or Lookup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_quantity">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_details_date_receiving">
        <property name="text">
         <string>Receiving Date:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_details_date_receiving</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_details_date_receiving"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_details_date_packed">
        <property name="text">
         <string>Packed On:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_details_date_packed</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_details_date_packed"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_details_date_expiration">
        <property name="text">
         <string>Expiration Date:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_details_date_expiration</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_details_date_expiration"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_details_quantity">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_details_quantity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Quantity:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_details_quantity</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Notes:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_details_notes">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_slot_warehouse</tabstop>
  <tabstop>lineEdit_slot</tabstop>
  <tabstop>lineEdit_product_sku</tabstop>
  <tabstop>pushButton_product_lookup</tabstop>
  <tabstop>lineEdit_product_description</tabstop>
  <tabstop>lineEdit_details_date_receiving</tabstop>
  <tabstop>lineEdit_details_date_packed</tabstop>
  <tabstop>lineEdit_details_date_expiration</tabstop>
  <tabstop>lineEdit_details_quantity</tabstop>
  <tabstop>lineEdit_details_notes</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
